src/components/Composer/index.js

```diff
@@ -337,52 +347,20 @@ class Composer extends React.Component {
             const domparser = new DOMParser();
             const embeddedImages = domparser.parseFromString(pastedHTML, TEXT_HTML).images;
 
-            // If HTML has img tag, then fetch images from it.
+            // Exclude parsing img tags in the HTML, as fetching the image via fetch triggers a Content-Security-Policy error.
```

Comment:
@qispark Looks like we have updated our CSP rule now to all fetching images from everywhere `img-src * blob: data:;`. Do you think this would now allow us to load the images from the pasted content correctly?

Reply:
I tested copying pasting content with images but even with CSP changes this doesn't work. Can you look into it so we update the comments correctly?

Reply:
@techievivek I was on Staging and still found a CSP issue as it currently stands. Can you let me know which environment has updated CSP and I can try and replicate?

Reply:
We have updated for both the platforms, you can confirm this by looking at the headers in response.

Reply:
OK. I have done some testing. The reason the CSP error is still occurring is related to `connect-src` rather than `img-src`. As we are trying to use the Fetch API, rather than fetching within an image tag, it relates to the `connect-src` policy.

Reply:
Aah, nice, that makas sense now. Thanks

Reply:
Can you please update the content to specify content-src CSP so it is easy to follow that in future.